org.learningequality.Kolibri.json

```diff
@@ -1,11 +1,13 @@
 {
   "app-id": "org.learningequality.Kolibri",
-  "runtime": "org.freedesktop.Platform",
-  "runtime-version": "19.08",
-  "sdk": "org.freedesktop.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
+  "sdk": "org.gnome.Sdk",
   "finish-args": [
     "--share=network",
-    "--persist=.kolibri"
+    "--persist=.kolibri",
+    "--socket=system-bus",
+    "--socket=session-bus"
```

Comment:
Please run the app with `--log-session-bus` or `--log-system-bus` to get the list of the DBus you should add here.

Reply:
Turns out neither was even necessary! I had assumed they would be needed for the D-Bus services I was adding (for search and activation), but it seem to be running fine without these permissions.